### PR TITLE
Shortcuts modal and Emoji picker UI Fixes

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
MM-24130 - Reducing spacing in keyboard shortcuts modal
MM-24159 - Updating emoji picker hover effect

@jasonblais Can you test this PR out on your screen, on my screen it looked good even previously, but reduced the spacing for you.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-24130
https://mattermost.atlassian.net/browse/MM-24159

#### Screenshots
localhost_8065_reiciendis-0_channels_eligendi-1
Screenshot 2020-04-15 at 2 00 56 PM

### CODE REVIEW

**Comment by jasonblais** [comment]
I can see the tip now, but the bottom of the modal gets cut off on the Mattermost window (same environment as in Jira ticket)

image

Compared to 5.21:
image

@asaadmahmood I realize it's because there are now a lot more shortcuts under "Navigation"

**Comment by asaadmahmood** [comment]
@jasonblais Yup, we have a lot more items, and we may have more in the future too, so it may not be possible to keep the modal visible completely at all times, especially on smaller resolutions.

**Comment by asaadmahmood** [comment]
@jasonblais I made it smaller on small screen sizes (i.e. height less than 800).

File: sass/routes/_shortcuts-modal.scss [hunk 1]
```diff
@@ -1,103 +1,119 @@
...
+                margin-top: 0;
             }
         }
     }
-}
+
+    .shortcut-line {
+        margin-top: 14px;
+    }
+}
```
Comment (marianunez) [inline sass/routes/_shortcuts-modal.scss:119]: nit: missing new line here

**Pull request merged**